src/main/scala/com/typesafe/conductr/sandbox/sbt/ConductRSandbox.scala

```diff
@@ -94,15 +96,36 @@ object ConductRSandbox extends AutoPlugin {
         val base = (Keys.baseDirectory in LocalRootProject).value
         val propFile = if (isMeta) base / TypesafePropertiesName else base / "project" / TypesafePropertiesName
         propFile.exists
-      }
+      },
+
+      // Internal keys
+      useDebugPort := false
     )
 
   override def projectSettings: Seq[Def.Setting[_]] =
     super.projectSettings ++ Seq(
       debugPort := 5005,
       // Here we try to detect what binary universe we exist inside, so we can
-      // accurately grab artifact reivisons.
-      isSbtBuild := Keys.sbtPlugin.?.value.getOrElse(false) && (Keys.baseDirectory in ThisProject).value.getName == "project"
+      // accurately grab artifact revisions.
+      isSbtBuild := Keys.sbtPlugin.?.value.getOrElse(false) && (Keys.baseDirectory in ThisProject).value.getName == "project",
+
+      BundleKeys.startCommand in Bundle :=
```

Comment:
The startCommand logic is quite complicated. Can we add documentation to explain the logic. This would make it simpler when looking at it the next time.